.php_cs.dist

```diff
@@ -47,7 +47,11 @@ $config = PhpCsFixer\Config::create()
         'single_line_comment_style' => true,
         'strict_comparison' => true,
         'strict_param' => true,
-        'yoda_style' => true,
+        'yoda_style' => [
+            'equal' => true,
+            'identical' => true,
+            'less_and_greater' => true,
+        ],
```

Comment:
does it mean we are more strict than `@Symfony` ruleset is ?

Reply:
yes, I didn't propose these options, only the default.

Reply:
ok, so it's more strict not because SF want to use default option, but because SF didn't make a decision what they want to use. thanks for clarification ;)

Reply:
yeah, that sounds about right. I never told them about the other options ;)

Reply:
then, maybe best approach would be to notify them, see if they want to actually adjust ruleset, and then use it as well?
to not have "our own way"

Reply:
@nicolas-grekas do you want, or know if SF community/maintainers want, to fix `greater`/`less` compares as well? (like the case you described @ https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/3075)

Reply:
I would, dunno about others.

Reply:
alrighty, thanks :)
against my own principles I'm going to ping some others in this case, @stof @fabpot if you've time and care about this topic please let us know, no rush and sorry for the ping, thanks!

Reply:
Yoda conditions are only useful for equality comparisons, where `==` could be inadvertently replaced by `=`. As conditions do not have this issue and because it makes the code less readable, I would not use Yoda conditions for comparisons.

Reply:
OK, I'll leave SF (the repo and the RuleSet) alone than.
I agree with Fabien here and would not apply the rule to this repo either (and thus not add it to the RuleSet of it). However that call is up to @keradus (@julienfalque do you've a preference?).

Reply:
IIRC, for each options we have `true|false|null`, like `yoda, non-yoda, ignore`.

In context of Symfony - if we don't want to enforce `yoda` for `less_and_greater`, maybe we want to enforce `non-yoda` for them?

Reply:
please don't enforce non-yoda, there is no reason to do so

Reply:
IMO comparisons with less/greater operator are harder to read with Yoda style so I would not apply it for such comparisons. Though I also agree with @nicolas-grekas, I would not force them to be non-Yoda either. Basically, I like the default options of the fixer :)

Reply:
so you said you find yoda for `>` hard to read, but you don't want to have non-yoda for them as well

Reply:
`if (123 < $a && $a < 256) {`
I'd like to be able to write this, yes.